### DISCUSSION TITLE
[Feature] Grey-out all other routes when one is clicked

### DISCUSSION THREAD

**Discussion by edbourque0** [johanohly/AirTrail#154] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would love that when I click on a route, all other routes shown on the map becomes grey-out or less opaque so that the map only shows this specific route.

**Comment by johanohly** [comment]
See my comment on #153

**Comment by johanohly** [comment]
This has been added in v2.0.0!
